tests/gravity_quadrature_degree_increase.prm

```diff
@@ -0,0 +1,73 @@
+# A simple setup for testing the gravity postprocessor using the
+# sample scheme 'list'.
+# See the corresponding section in the manual for more information.
+
+# General parameters
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output-gravity_quadrature_degree_increase.prm
+set Nonlinear solver scheme                = no Advection, no Stokes
+
+# Model geometry
+subsection Geometry model
+  set Model name = spherical shell
+  subsection Spherical shell
+    set Inner radius  = 1
+    set Outer radius  = 2
+    set Cells along circumference = 12
+  end
+end
+
+# Model boundary velocity
+subsection Boundary velocity model
+  set Zero velocity boundary indicators       = top, bottom
+end
+
+# Material model
+subsection Material model
+  set Model name = simple
+  subsection Simple model
+    set Reference density                 = 1e6
+  end
+end
+
+# Model boundary temperature
```

Comment:
Can you remove the "Boundary temperature model" section? It is currently ignored by ASPECT anyway, and will soon throw an error (when #4162 is merged).

Reply:
hopefully correctly addressed all points